client/src/components/History/HistoryDetails.vue

```diff
@@ -128,8 +130,8 @@
             class="history-title mt-2"
             tag-name="h3"
             v-model="historyName"
-            :tooltip-title="'Rename history...' | localize"
-            tooltip-placement="top"
+            :title="'Rename history...' | localize"
+            v-b-tooltip.hover="{ boundary: 'viewport', placement: 'left' }"
```

Comment:
I don't care if it is top or left, but annotations tooltip on the left and without the other part of the change the tooltip doesn't work at all.